torch/_prims/nvfuser_executor.py

```diff
@@ -21,7 +21,6 @@
 if torch.cuda.is_available():
     from nvfuser._C import (  # type: ignore[import]
         DataType,
-        Fusion,
```

Comment:
hmmm.. Now I realize that we have primTorch stuff using nvfuser python API, this will be ugly pushing changes to upstream for these.

Reply:
I think I caught all the necessary changes, what makes this ugly?  Is it just because you have to get approval from upstream?

Reply:
Yep. changes going to folder outside of `./third_party/nvfuser` is... unfortunate. :smile: